server/scsynth/SC_World.cpp

```diff
@@ -351,14 +351,12 @@ World* World_New(WorldOptions *inOptions)
 		HiddenWorld *hw = world->hw;
 		hw->mGraphDefLib = new HashTable<struct GraphDef, Malloc>(&gMalloc, inOptions->mMaxGraphDefs, false);
 		hw->mNodeLib = new IntHashTable<Node, AllocPool>(hw->mAllocPool, inOptions->mMaxNodes, false);
-		hw->mUsers = (ReplyAddress*)zalloc(inOptions->mMaxLogins, sizeof(ReplyAddress));
-		hw->mNumUsers = 0;
+		hw->mUsers = new Clients();
 		hw->mMaxUsers = inOptions->mMaxLogins;
-		hw->mClientIDs = (uint32*)zalloc(inOptions->mMaxLogins, sizeof(uint32));
+		hw->mAvailableClientIDs = new ClientIDs();
 		for (int i = 0; i<hw->mMaxUsers; i++) {
```

Comment:
Should this be ```for (auto addr : hw->mUsers)``` to match the others?

Reply:
no, this loop is appending integers, not addresses